src/worldfactory.cpp

```diff
@@ -1927,12 +1908,6 @@ int worldfactory::show_worldgen_basic( WORLD *world )
                 int lvl = wg_slevels[sel_opt - 1] + ( action == "LEFT" ? -1 : 1 );
                 wg_slevels[sel_opt - 1] = clamp<int>( lvl, 0, wg_sliders[sel_opt - 1]->count() - 1 );
                 wg_sliders[sel_opt - 1]->apply_opts( wg_slevels[sel_opt - 1], world->WORLD_OPTIONS );
-            } else if( sel_opt > static_cast<int>( wg_sliders.size() ) ) {
```

Comment:
worldgen option "buttons" are displayed vertically now, so don't need this anymore